apps/web/src/pages/chat/page.tsx

```
@@ -251,17 +251,23 @@ export default function ChatPage() {
           });
           refreshCredits();
         });
+
+        // Load history once connection is established
+        loadHistory();
       })
       .catch((err) => console.error('SignalR start error:', err));
-  }, [connection, addMessage]);
+  }, [connection, addMessage, loadHistory]);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)